opencog/openpsi/utilities.scm

```diff
@@ -326,13 +326,13 @@
 		result
 		; else check if entity is an evaluation or predicate or schema
 		(let ((type (cog-type entity)))
-			(if (or (equal? type 'PredicateNode)
-				   (equal? type 'GroundedPredicateNode)
+			(if (or (equal? type 'GroundedPredicateNode)
 				   (equal? type 'DefinedPredicateNode))
 				(set! result (cog-evaluate! (Evaluation entity (List)))))
-			(if (or (equal? type 'SchemaNode)
-            		(equal? type 'GroundedSchemaNode)
-            	    (equal? type 'DefinedSchemaeNode))
+			(if (equal? type 'PredicateNode)
+				(set! result (cog-tv entity)))
```

Comment:
Well since they are both valid representations with no meaningful difference, what if one process, e.g., ros rig control, uses the PredicateNode TV to store the value, and another process, e.g., PLN, uses an EvaluationLink TV to store the value. Neither "user" has made an error, and we now have different values.